andela/kissa-cfh-project
# Chore/134434829/GitHub pivotal tracker integration

#### What does this PR do?
This PR integrates GitHub with Pivotal tracker, such that all commit messages tagged with a user story Id will be added to the story's activity log.

#### Description of Task completed?
Added an API token from Pivotal Tracker to GitHub.

#### How should this be manually tested?
Make a commit message and include the user story ID in square braces, then check you pivotal tracker story activity log to find the commit messages
e.g 	`feature(facebook-like): add facebook iframe to index.html **[Start #134678595]**`

#### Any background context you want to provide?
No background context

#### What are the relevant pivotal tracker stories?
Integrate Github commits with Pivotal Tracker #134434829

#### Screenshots (if appropriate)
![new scrr](https://cloud.githubusercontent.com/assets/23451840/21059666/e8b62330-be43-11e6-92f6-711dc925f414.png)

#### Questions:
No questions.
